providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/operators/job.py

```diff
@@ -101,6 +101,7 @@ def __init__(
         wait_until_job_complete: bool = False,
         job_poll_interval: float = 10,
         deferrable: bool = conf.getboolean("operators", "default_deferrable", fallback=False),
+        on_kill_propagation_policy: str = "Background",
```

Comment:
the old behavior is to call `super().onkill()` to delete the pod, which seems to be the same behavior as setting the propagation policy to "Foreground", i.e. immediately delete pods that belong to the job.

Reply:
please also update the doc string to include this new field

Reply:
Hello @steinwaywhw thank you for your review! Here I totally agree with you. Set the policy to Foreground and add updated doc string